### PR TITLE
update README

### PR DESCRIPTION
**Pull request by Incand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+- Open issue with suggetion for change
+- No one complains -> submit pull request with changes
+- Important! Concurrently working on the same map is very problematic
+  - Therefore, changes should be as small as possible and quickly transition from issue to PR
+
+## Working with Tiled
+
+Tiled: Intuitive tool for creating tilemaps
+
+### Tilesets
```
Comment (priegger) [inline README.md:24]: Todo (separate commit?)

We should probably add some metadata, where we got the file from and how it is licensed.

File: README.md [hunk 2]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - Atlasse used as a register for tiles to draw
+  - For workadventure, tiles should be 32 x 32 pixel
+  - Add a tileset via import of a png
```
Comment (priegger) [inline README.md:27]:
```suggestion
  - Add a tileset via import of a png (this has to be done separately for each map where the tileset is used)
```

File: README.md [hunk 3]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - Important! Tilesets have to be embedded into the tilemap JSONs
+
+### Tilemap
+  - A map in which tiles from one or more tilesets are placed
+  - Multiple layers
+  - Important! Has to be exported as JSON
+
+### Advanced
```
Comment (priegger) [inline README.md:35]: MAybe add links for the advanced topics? Or a "teaser" sentence on why one might want to look into the topic?

File: README.md [hunk 4]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - Animated Tiles
+  - TerrainSets
+
+### WorkAdventure Integration
```
Comment (priegger) [inline README.md:39]: I'd directly add an introductory sentence to this section with a link to the map-building tutorial. People should see/use that link first. What we have here is just a summary/teaser.

File: README.md [hunk 5]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - "floorLayer" ObjectLayer is needed (rendering of characters happens here)
+  - "start" layer used as the default entrypoint
+  - Optinally jitsii-meeting layers for conferencing
```
Comment (priegger) [inline README.md:42]: Todo (new commit) we should add a section about the jitsi meet names, namespacing, etc.

```suggestion
  - Optinally jitsii-meeting layers for conferencing (one layer per meeting)
```

File: README.md [hunk 6]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - Map currently in work can be tested with referencing their GitHub raw-links in the URL
+    - e.g.: https://workadventure.itemis.de/_/nord/raw.githubusercontent.com/itemis-Nord/ces-workadventure-maps/master/cloud-hub.json
```
Comment (priegger) [inline README.md:44]: I'll rebase #17 on this change and extend this example. Thanks for getting the README started!

File: README.md [hunk 7]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+  - "floorLayer" ObjectLayer is needed (rendering of characters happens here)
+  - "start" layer used as the default entrypoint
+  - Optinally jitsii-meeting layers for conferencing
+  - Map currently in work can be tested with referencing their GitHub raw-links in the URL
+    - e.g.: https://workadventure.itemis.de/_/nord/raw.githubusercontent.com/itemis-Nord/ces-workadventure-maps/master/cloud-hub.json 
+  
+## Ressources
+
+- WorkAdventure Map-Building Tutorial: [https://workadventu.re/map-building](https://workadventu.re/map-building)
+- Tiled Editor: [https://www.mapeditor.org/](https://www.mapeditor.org/)
```
Comment (priegger) [inline README.md:49]: Why not like this?
```suggestion
- [WorkAdventure Map-Building Tutorial](https://workadventu.re/map-building)
- [Tiled Map Editor](https://www.mapeditor.org/)
```

File: README.md [hunk 8]
```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
...
+- No one complains -> submit pull request with changes
+- Important! Concurrently working on the same map is very problematic
+  - Therefore, changes should be as small as possible and quickly transition from issue to PR
+
+## Working with Tiled
+
+Tiled: Intuitive tool for creating tilemaps
+
+### Tilesets
+  - Atlasse used as a register for tiles to draw
```
[the remark this answers is not in the document]
Comment (Incand) [reply README.md:25]: https://duckduckgo.com/?q=Sprite+Atlases&t=ffab&ia=web

File: README.md (hunk 1 above)
Comment (Incand) [reply README.md:24]: What do you mean by that?

Comment (priegger) [reply README.md:25]: Something like this:          https://github.com/itemis-Nord/ces-workadventure-maps/blob/8e052257584937b6ba878e275587c829c4463f57/map.json#L1002

I think it makes sense to make it mandatory.

**Review by priegger — APPROVED** [review]

**Pull request merged**